docs/hive-integration/kube-api-install-retry.md

```diff
@@ -0,0 +1,141 @@
+# Hive Integration - Retry to Install a cluster
+
+In a case where the cluster installation has failed, the user may want to restart the installation.
+A failed install may happen for multiple reasons; make sure you understand the root cause before trying again.
+
+To restart the installation, the user will need to:
+1. Delete the cluster `AgentClusterInstall` resource.
+2. Delete all the cluster `BareMetalHost` resources (a single resource in the case of SNO).
+3. Create the cluster `AgentClusterInstall` resource again.
+4. Create all the cluster `BareMetalHost` resources (a single resource in the case of SNO).
```

Comment:
What do you think about rephrasing tihs to something like:

Recreate the BMH resources that were deleted in step 2

Reply:
done